src/cobra/test/test_util/test_context.py

```diff
@@ -0,0 +1,42 @@
+"""Test functions of context.py."""
+
+import pytest
+
+from cobra.util import HistoryManager, get_context, resettable
+
+
+def test_history_manager():
+    """Test initialization and resetting of HistoryManager."""
+    # initialize HistoryManager
+    history_manager = HistoryManager()
+    # add non-functioning operation
+    history_manager(lambda: 1)
+    # reset operations
+    history_manager.reset()
+
+
+def test_get_context(model):
```

Comment:
```suggestion
def test_get_context(model: cobra.Model) -> None:
```

Reply:
I think using `"Model"` with `TYPE_CHECKING` check is a bit cleaner.